src/config/generalconf.cpp

```diff
@@ -264,6 +270,17 @@ void GeneralConf::initSaveLastRegion()
             this,
             &GeneralConf::saveLastRegion);
 }
+void GeneralConf::initRandomizePixelation()
```

Comment:
@kurayami07734 You never call this method.

Reply:
all init methods are called once in the constructor method at the beginning of this file